src/utils/signedUrlReplacer.ts

```diff
@@ -2,12 +2,13 @@ import { Prisma, PrismaClient, StateType, WorkflowState } from '@prisma/client'
 import { DefaultArgs } from '@prisma/client/runtime/library'
 
 export async function replaceImageSrc(htmlString: string, getSignedUrl: (filePath: string) => Promise<string | undefined>) {
-  const imgTagRegex = /<img\s+[^>]*src="([^"]+)"[^>]*>/g //expression used to match all img tags in provided HTML string.
+  const imgTagRegex = /<img\s+[^>]*src="([^"]+)"[^>]*>/g
+  const attachmentTagRegex = /data-type="attachment"\s+[^>]*src="([^"]+)"[^>]*>/g //expression used to match all img tags in provided HTML string.
```

Comment:
this comment is incorrect, this regex is different. Also where does data-type get set? We should update this comment to explain _why_ this regex works

Reply:
Does this regex need to match any attachment or just the image? In which case, should the src matter? Is it the case that we only have image attachment? Can we include mime type in our attachment extension to make things easier for us maybe?

Reply:
This change needs to be reverted because I realized we do not need fresh signed urls for attachments. Unlike images, attachments needs to be fetched on demand only(download) and not every time. So, doing this was unnecessary.